t/case.neon

```diff
@@ -1,6 +1,5 @@
-VAR a: Number
-VAR s: String
-a := 0
+VAR a: Number := 0
+VAR s: String := ""
```

Comment:
Why did you change this initialization? `a` is fine, but `s` was *specifically* uninitialized. Please keep it that way.